core/domain/issue_registry.py

```diff
@@ -35,7 +35,7 @@ def get_all_issue_ids(cls):
         Returns:
             list(str). The list of all allowed issue IDs.
         """
-        return feconf.ALLOWED_ISSUE_IDS
+        return feconf.ALLOWED_ISSUE_TYPES
```

Comment:
docstring says ids, this says types, please standardize.

Reply:
Done.